src/plugins/data_explorer/public/components/app_container.scss

```diff
@@ -0,0 +1,18 @@
+$osdHeaderOffset: $euiHeaderHeightCompensation;
+
+.deSidebar {
+  max-width: 462px;
+  min-width: 400px;
+}
+
+.deLayout {
+  height: calc(100vh - #{$osdHeaderOffset});
+
+  &__canvas {
+    height: 100%;
+  }
+}
+
+.headerIsExpanded .deLayout {
+  height: calc(100vh - #{$osdHeaderOffset * 2});
```

Comment:
Is this always true? irrespective of the branding assets being loaded?

Reply:
Yes this is the same calc i used for VisBuilder. This works because the height of the top nav is fixed.